internal/pkg/pct/build.go

```diff
@@ -57,3 +67,36 @@ func (b *Builder) Build(templatePath, targetDir string) (gzipArchiveFilePath str
 
 	return gzipArchiveFilePath, nil
 }
+
+func (b *Builder) checkConfig(configFile string) error {
```

Comment:
Actually, thinking about this more and looking at #181  , since we need to parse and check the config, should this be moved to an internal package? Or is it fine to reimplement the unmarshal/validate there?